modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -13,6 +13,9 @@ Install and use the `rtfctl` command line tool to locally manage Runtime Fabrics
 * Manage a Mule license.
 * Manage the alert sender email address.
 
+[Note]
+If you use the rtf namespace,you do not need to specify an additional `--namespace <rtfNamespace>`
```

Comment:
```suggestion
If you are using the default namespace (rtf), you do not need to specify the additional flag: `--namespace <rtfNamespace>`
```

Reply:
Is a note enough? Do we have a way to put this in a box or something to make it more obvious?

Reply:
Please also note that this flag is only introduced as of rtfctl version >= 0.4.0